docs/topics/installing-vs-code-extension.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * docs/vsc-extension-guide/master.adoc
+
+
+[id="installing-vs-code-extension_{context}"]
+= Installing the {ProductShortName} extension on Visual Studio Code
+
+You can install the {ProductShortName} extension on Visual Studio Code (VS Code) on Linux, Windows, or macOS.
+
+.Prerequisites
+
+* link:https://code.visualstudio.com[VS Code installed]
+* 8 GB RAM
+* JDK 11 or later
```

Comment:
Spell it out in full the first time. Also, do the same for the other installations.